### ISSUE TITLE
Twilio Support

### ISSUE THREAD

**Issue by AlhasanIQ** [appleboy/gorush#576]
I am willing to add Support for Twilio SMS/Whatsapp if it is considered in-scope of the project.

**Comment by genofire** [comment]
I do not believe it is in-scope - this is an Service for Webplatform, to send PushNotifications to the (own develop) App. So that this App knows, that something happen on the Webplatform.

The Scope is, the inform own developing Apps. So Whatsapp-App has his own PushNotification Service, that you have new Message on Whatsapp-Platform.

**Comment by AlhasanIQ** [comment]
I don't believe that what you said is particularly accurate. 

The reason I suggested Twilio (sms,whatsapp,sendgrid, ...etc) is the way i think of this project, as a "toolkit"-like app to manage my "notifications". Most modern webservices send sms'es, emails, and push notifications. And it would be relatively trivial to add this much needed functionality to gorush (as drivers/adapters).

Doesn't even have to be a `batteries-included` solution. Could simply be an extension/add-on.
